uid/snowid_test.go

```diff
@@ -331,10 +329,31 @@ func FuzzID_Parse_RoundTrip_FromString(f *testing.F) {
 	})
 }
 
+func TestNormalizesLeadingZeroValue(t *testing.T) {
```

Comment:
I always try to be explicit about which function or method is being tested in the name, to make it easier to find the tests for a particular function

```suggestion
func TestParse_NormalizesLeadingZeroValue(t *testing.T) {
```